pommerman/envs/v0.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
         self._agents = None
         self._game_type = game_type
         self._board_size = board_size
+        self._mode = None
```

Comment:
set self._mode = 'human' here as a default and add a comment that it can be changed with set_render_mode below.

Reply:
Cool.